addons/mail/models/ir_attachment.py

```diff
@@ -71,5 +82,90 @@ def _attachment_format(self, commands=False):
                     'res_id': attachment.res_id,
                     'res_model': attachment.res_model,
                 })
-            res_list.append(res)
-        return res_list
+            attachments.append(res)
+        return attachments
+
+    @api.model
+    def _get_link_preview_from_url(self, url):
+        try:
+            page = requests.get(url, timeout=1)
```

Comment:
This timeout seems a bit low if it has to download an image/a rather large gif, or just if there is any network slowness.

Reply:
It was asked to be 1s inside a review of an other iteration of this PR.
I'll update to 3s